setup.py

```diff
@@ -25,7 +25,7 @@
     "django-taggit>=4.0,<5.1",
     "django-treebeard>=4.5.1,<5.0",
     "djangorestframework>=3.15.1,<4.0",
-    "django-filter>=23.3,<24",
+    "django-filter>=23.3,<=24.2",
```

Comment:
```suggestion
    "django-filter>=23.3,<25",
```

So that we can be more flexible for updates to this Library we should follow the current convention of allowing all versions in the next major.